src/api/app/assets/javascripts/webui2/attributes.js

```diff
@@ -0,0 +1,9 @@
+$(document).ready(function() {
+  // TODO: Edge case - When a user is already on the page and does a refresh, the description flickers as it appears. Find a way to fix this...
+  $('p#attribute_type-description-' + $('select#attrib_attrib_type_id option:selected').val()).removeClass('d-none');
```

Comment:
Since you already use ids as selector you can just use the, eg. #attrib_attrib_type_id instead of select#attrib_attrib_type_id